modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -4,123 +4,68 @@ include::_attributes.adoc[]
-* Proxies are not versioned in Mule 3.x applications.
-* You can edit proxies from API Manager for Mule versions 3.8.x and later.
+You can download a preconfigured Mule application, if you are running an on-premise Mule instance, and deploy it just as any other Mule application by simply xref:org-credentials-config-mule4.adoc[configuring your Mule instance with the correct Anypoint Platform credential].
+ 
+The following diagram illustrates the how a request communicates with the proxy endpoint and returns a response from the backend sever:
 
-[[default-setting-mule-4]]
-==== Default Setting by API Proxy Versions for Mule 4.x Applications
-
-[%header%autowidth.spread]
-|===
-| Mule Version | API Proxy | Default Setting | UI Support
-| 4.x| RAML 1.1.x            | Enabled  | No
-| 4.x| RAML 1.2.x            | Enabled  | Yes
-| 4.x| REST 2.0.x            | Enabled  | No
-| 4.x| REST 2.1.0 or later  | Enabled   | Yes
-| 4.x| WSDL 1.2.0 and earlier  | Disabled | No
-| 4.x| WSDL 1.2.1 and later  | Disabled | Yes
-|===
-
-Mule 4.x applications and API proxy compatibility synopsis:
-
-* For Mule 4.x applications, both regular and strict validations are available.
-* Validations are performed for only WSDL and RAML or REST proxies.
-* Proxies are versioned in Mule 4.x applications. 
-* You can edit RAML or REST proxies from API Manager for RAML proxy version 1.2.0 and REST proxy version 2.1.0 and later.
-
-
-// === API Specifications for Strict Validations
-// When enabling strict validations for Mule 4 applications, you must consider your API architecture. If your Mule application has a proxy or a load balancer in front of it, the web server might modify the client request to include additional headers. In such cases, the Mule application denies the incoming request due to a non-valid header error.
-
-// Note that the newer versions of CloudHub and RuntimeFabric also include headers when sending requests to the API.
-
-// ==== Runtime Fabric headers
-
-// The following headers are added by Edge in Runtime Fabric:
-// [source,RAML,linenums]
-// ----
-// headers:
-//   x-correlation-id:
-//     type: string
-//   x-forwarded-for:
-//     type: string
-//   via:
-//     type: string
-//   x-forwarded-host:
-//     type: string
-// ----
-
-// Therefore, consider adding these headers in the API specification when using Edge.
+image::api-gateway-capabilities-mule4-62bf9.png[align=center]
+ 
+As shown in the diagram, when an external consumer application sends a request, it first pings the proxy endpoint. At this time, all policies applied to the API that are referenced in the proxy application takes effect. 
+
+If a policy fails any of the validations, an error response is returned (appropriate to the failed policy) and the request does not reach your backend. Conversely, if all the validations and actions performed by the policies are successful, the request continues to your backend API, and the proxy then returns this response to the consumer application.
```

Comment:
"response is returned" is redundant.

Reply:
"appropriate to the failed policy" is unnecessary because the sentence begins with "if a policy fails."

Reply:
```suggestion
If a policy fails any of the validations, the request does not reach your backend API. Conversely, if all the validations are successful, the request continues to your backend API for processing, and the proxy enables the response to return to the consumer application.
```

Reply:
I wonder if it is misleading to say "a policy fails the validations"? The policy is not what fails, right? The policy is what protects the backend from invalid requests? So, "if the policy discovers an invalid request"?

Reply:
Done! Thanks!